feature/topic/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/topic/TopicScreen.kt

```diff
@@ -280,13 +286,18 @@ private fun TopicToolbar(
             .fillMaxWidth()
             .padding(bottom = 32.dp),
     ) {
-        IconButton(onClick = { onBackClick() }) {
-            Icon(
-                imageVector = NiaIcons.ArrowBack,
-                contentDescription = stringResource(
-                    id = com.google.samples.apps.nowinandroid.core.ui.R.string.core_ui_back,
-                ),
-            )
+        if (showBackButton) {
```

Comment:
Nit: Could be fun to animate the visibility here.

Reply:
We can explore that. My initial impression is that it wouldn't be particularly noticeable though, because the animation would only be relevant if there's a config change that changes how many panes are visible.